src/averageAges.js

```diff
@@ -37,7 +47,12 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  return people
+    .filter(person => (person.sex === 'f'))
+    .filter((person) => withChildren ? hasChildren(people, person) : true)
+    .reduce((average, person, index, men) => {
```

Comment:
why are you accepting `men` as parameter?

Reply:
Because I need to get the array length here to calculate average but it would be better to name it women here :D